clang/lib/AST/ExprConstant.cpp

```diff
@@ -7011,6 +7023,12 @@ class APValueToBufferConverter {
     return visitInt(AsInt, Ty, Offset);
   }
 
+  bool visitDecimalFloat(const APDecimalFloat &Val, QualType Ty,
+                         CharUnits Offset) {
+    APSInt AsInt(Val.bitcastToAPInt());
+    return visitInt(AsInt, Ty, Offset);
```

Comment:
Do we need to worry about the availability of 128-bit integer support here? I don't think that concern exists for fixed-point types.